datadog_lambda/tracing.py

```diff
@@ -689,10 +874,20 @@ def create_inferred_span_from_http_api_event(event, context):
         "span_type": "http",
     }
     tracer.set_tags({"_dd.origin": "lambda"})
+    inferred_span_start_ns = None
+    if decode_authorizer_context:
```

Comment:
Can we break this up into a method to simplify this conditional?
Then we could do `if injected_authorizer_data` here, and in the method, we could check if we should decode context.

Reply:
I further cleaned up this a bit. However, I do want keep the `if decode_authorizer_context:` at this scope so that we can see if it's False we will just skip completely.